multicluster/cmd/install.go

```diff
@@ -29,6 +29,8 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+var ignoreCluster bool
```

Comment:
Can this move into `newMulticlusterInstallCommand` similar to other flag variables that are declared there?

Reply:
Done in https://github.com/linkerd/linkerd2/pull/7564/commits/747061fdca1ebf0626a2edb93babfdb804c51f67